src/blocks/faq/edit.js

```diff
@@ -56,6 +56,7 @@ const Edit = ( props ) => {
 			<div className={ className }>
 				<InnerBlocks
 					allowedBlocks={ ALLOWED_BLOCKS }
+					placeholder={ ( <em>{ __( 'Click here to add an item to the FAQ', 'coblocks' ) }</em> ) }
```

Comment:
This is a cleaner solution than removing blocks. Strange that the other behavior is to show nothing at all. 🤷